### ISSUE TITLE
Station ownerships

### ISSUE THREAD

**Issue by Allalous** [RobertSkalko/Mine-and-Slash#355]
I've been playing with this mod on the Craft to Ecile 2 pack and after I place down any of the stations (cooking, salvaging ect) my friends get a message saying "You do not have ownership". I can't find a command or anything on how to fix this, other than we just have everyone make their own and place them down, which makes everything look so cluttered. Is that the only fix or did I miss something?

**Comment by confuseduser2** [comment]
Hi I've ran into the same issue did you find a fix?

**Comment by Allalous** [comment]
No, we have not found a fix other than just having all the tables and things just in their own corners with chests and hoppers so everyone can still get the stuff from them
